addons/sourcemod/scripting/MyJailbreak/playertags.sp

```diff
@@ -268,40 +268,50 @@ void LoadPlayerTags(int client)
 	}
 
 	// Check SteamID again with bad steam universe
-	if (StrContains(steamid64, "STEAM_0", false) != -1)
+	if (StrContains(steamid, "STEAM_0", false) != -1)
 	{
-		ReplaceString(steamid64, sizeof(steamid64), "STEAM_0", "STEAM_1", false);
+		ReplaceString(steamid, sizeof(steamid), "STEAM_0", "STEAM_1", false);
 	}
-	else ReplaceString(steamid64, sizeof(steamid64), "STEAM_1", "STEAM_0", false);
+	else ReplaceString(steamid, sizeof(steamid), "STEAM_1", "STEAM_0", false);
 
-	if (kvMenu.JumpToKey(steamid64, false))
+	if (kvMenu.JumpToKey(steamid, false))
 	{
 		GetTags(client, kvMenu);
 
 		delete kvMenu;
 		return;
 	}
 
-	// Check flags
-	char flags[32];
-	GetAdminFlagsEx(client, flags);
-	LogMessage("Flags: %s", flags);
-	// backwards loop
-	for (int i = strlen(flags)-1; i >= 0 ; i--)
+	// Check groups
+	AdminId admin = GetUserAdmin(client);
+	if (admin != INVALID_ADMIN_ID)
 	{
-		LogMessage("Flags vor buffer: %s", flags[i]);
-		char buffer[2];
-		Format(buffer, sizeof(buffer), flags[i]);
-		LogMessage("Flags nach buffer: %s", buffer);
-
-		if (kvMenu.JumpToKey(buffer, false))
+		char sGroup[32];
+		admin.GetGroup(0, sGroup, sizeof(sGroup));
```

Comment:
Not sure if 0 it's ok or if it's needed to loop [GroupCount](https://sm.alliedmods.net/new-api/admin/AdminId/GroupCount) times.

Reply:
When I understand description on API docs right, this int is a wildcard.
if the 0 should not work maybe use it with a variable. I don't think we need the GroupCount here.
```
int iBuffer;
admin.GetGroup(iBuffer, sGroup, sizeof(sGroup));
```

Reply:
I've already used this function on other plugins, and there I used "0", from what I understood "0" is the index of the group, like if the client is in the group *Mod* and in the group *Admin*, the index of one group will be 0 and of the second group would be 1.
I mean something like:
```
	...
	for (int i = 0; i <= admin.GroupCount)
	{
		char sGroup[32];
		admin.GetGroup(i, sGroup, sizeof(sGroup));
		...
	}
```

Reply:
uhhmm... true. I guess I must still be on medication.

Yeah, a loop is needed the player can be in more than one group. would be interesting to know how the group index is determined. the order in cfg or alphabetical.

I never worked with groups so far.